--- /etc/glance/glance-registry.conf.old        2015-09-08 17:44:38.308330561 +0000
+++ /etc/glance/glance-registry.conf    2015-09-08 20:24:00.556986330 +0000
@@ -1,6 +1,6 @@
 [DEFAULT]
 # Show more verbose log output (sets INFO log level output)
-#verbose = False
+verbose = True
 
 # Show debugging output in logs (sets DEBUG log level output)
 #debug = False
@@ -91,7 +91,7 @@
 # Deprecated group/name - [DEFAULT]/sql_connection
 # Deprecated group/name - [DATABASE]/sql_connection
 # Deprecated group/name - [sql]/connection
-#connection = <None>
+connection = mysql://glance:password@ops-mono-node/glancedb
 
 # The SQL mode to be used for MySQL sessions. This option,
 # including the default, overrides any server-set SQL mode. To
@@ -172,12 +172,12 @@
 #db_max_retries = 20
 
 [keystone_authtoken]
-auth_host = 127.0.0.1
+auth_host = ops-mono-node
 auth_port = 35357
 auth_protocol = http
-admin_tenant_name = %SERVICE_TENANT_NAME%
-admin_user = %SERVICE_USER%
-admin_password = %SERVICE_PASSWORD%
+admin_tenant_name = service
+admin_user = glance
+admin_password = password
 
 [paste_deploy]
 # Name of the paste configuration file that defines the available pipelines
@@ -187,4 +187,4 @@
 # service name removed. For example, if your paste section name is
 # [pipeline:glance-registry-keystone], you would configure the flavor below
 # as 'keystone'.
-#flavor=
+flavor=keystone
