--- /etc/neutron/plugins/ml2/ml2_conf.ini.bak	2016-06-10 10:32:11.753118716 -0400
+++ /etc/neutron/plugins/ml2/ml2_conf.ini	2016-06-10 10:41:37.625112642 -0400
@@ -104,16 +104,16 @@
 
 # List of network type driver entrypoints to be loaded from the
 # neutron.ml2.type_drivers namespace. (list value)
-type_drivers = flat,vlan
+type_drivers = flat,vlan,gre,vxlan
 
 # Ordered list of network_types to allocate as tenant networks. The default
 # value 'local' is useful for single-box testing but provides no connectivity
 # between hosts. (list value)
-tenant_network_types = 
+tenant_network_types = vxlan
 
 # An ordered list of networking mechanism driver entrypoints to be loaded from
 # the neutron.ml2.mechanism_drivers namespace. (list value)
-mechanism_drivers = linuxbridge
+mechanism_drivers = opendaylight
 
 # An ordered list of extension driver entrypoints to be loaded from the
 # neutron.ml2.extension_drivers namespace. For example: extension_drivers =
@@ -183,7 +183,7 @@
 
 # Comma-separated list of <tun_min>:<tun_max> tuples enumerating ranges of GRE
 # tunnel IDs that are available for tenant network allocation (list value)
-#tunnel_id_ranges =
+tunnel_id_ranges = 1:1000
 
 
 [ml2_type_vlan]
@@ -207,7 +207,7 @@
 
 # Comma-separated list of <vni_min>:<vni_max> tuples enumerating ranges of
 # VXLAN VNI IDs that are available for tenant network allocation (list value)
-#vni_ranges =
+vni_ranges = 1001:2000
 
 # Multicast group for VXLAN. When configured, will enable sending all broadcast
 # traffic to this multicast group. When left unconfigured, will disable
@@ -232,3 +232,10 @@
 # Use ipset to speed-up the iptables based security groups. Enabling ipset
 # support requires that ipset is installed on L2 agent node. (boolean value)
 enable_ipset = true
+enable_security_group = True
+enable_ipset = True
+firewall_driver = neutron.agent.not.a.real.FirewallDriver
+[ml2_odl]
+password = admin
+username = admin
+url = http://192.168.33.20:8080/controller/nb/v2/neutron
