--- /etc/neutron/plugins/ml2/ml2_conf.ini.odl	2016-05-24 09:06:33.047071250 +0000
+++ /etc/neutron/plugins/ml2/ml2_conf.ini	2016-05-24 09:10:06.793896341 +0000
@@ -2,20 +2,20 @@
 # (ListOpt) List of network type driver entrypoints to be loaded from
 # the neutron.ml2.type_drivers namespace.
 #
-type_drivers = flat,vlan
+type_drivers = flat,vlan,gre,vxlan
 # Example: type_drivers = flat,vlan,gre,vxlan,geneve
 
 # (ListOpt) Ordered list of network_types to allocate as tenant
 # networks. The default value 'local' is useful for single-box testing
 # but provides no connectivity between hosts.
 #
-tenant_network_types = 
+tenant_network_types = vxlan
 # Example: tenant_network_types = vlan,gre,vxlan,geneve
 
 
 # (ListOpt) Ordered list of networking mechanism driver entrypoints
 # to be loaded from the neutron.ml2.mechanism_drivers namespace.
-mechanism_drivers = linuxbridge
+mechanism_drivers = opendaylight
 # Example: mechanism_drivers = openvswitch,mlnx
 # Example: mechanism_drivers = arista
 # Example: mechanism_drivers = openvswitch,cisco_nexus,logger
@@ -64,7 +64,7 @@
 # can be created. Use * to allow flat networks with arbitrary
 # physical_network names.
 #
-flat_networks = public
+flat_networks = external
 # Example:flat_networks = physnet1,physnet2
 # Example:flat_networks = *
 
@@ -79,13 +79,13 @@
 
 [ml2_type_gre]
 # (ListOpt) Comma-separated list of <tun_min>:<tun_max> tuples enumerating ranges of GRE tunnel IDs that are available for tenant network allocation
-# tunnel_id_ranges =
+tunnel_id_ranges = 1:1000
 
 [ml2_type_vxlan]
 # (ListOpt) Comma-separated list of <vni_min>:<vni_max> tuples enumerating
 # ranges of VXLAN VNI IDs that are available for tenant network allocation.
 #
-# vni_ranges =
+vni_ranges = 1001:2000
 
 # (StrOpt) Multicast group for the VXLAN interface. When configured, will
 # enable sending all broadcast traffic to this multicast group. When left
@@ -113,8 +113,15 @@
 [securitygroup]
 # Controls if neutron security group is enabled or not.
 # It should be false when you use nova security group.
-# enable_security_group = True
+enable_security_group = True
+firewall_driver = neutron.agent.not.a.real.FirewallDriver
 
 # Use ipset to speed-up the iptables security groups. Enabling ipset support
 # requires that ipset is installed on L2 agent node.
 enable_ipset = True
+
+[ml2_odl]
+password = admin
+username = admin
+url = http://127.0.0.1:8188/controller/nb/v2/neutron
+
