--- /home/vagrant/distribution-karaf-0.4.1-Beryllium-SR1/etc/org.apache.karaf.features.cfg.old	2016-05-24 12:15:21.423233180 +0000
+++ /home/vagrant/distribution-karaf-0.4.1-Beryllium-SR1/etc/org.apache.karaf.features.cfg	2016-05-24 12:16:13.463235839 +0000
@@ -41,7 +41,7 @@
 #
 # Comma separated list of features to install at startup
 #
-featuresBoot=config,standard,region,package,kar,ssh,management
+featuresBoot=config,standard,region,package,kar,ssh,management,odl-ovsdb-openstack
 
 #
 # Defines if the boot features are started in asynchronous mode (in a dedicated thread)
