--- /etc/neutron/neutron.conf.old	2016-02-24 13:28:30.000000000 +0000
+++ /etc/neutron/neutron.conf	2016-04-04 14:57:04.221241701 +0000
@@ -1,6 +1,6 @@
 [DEFAULT]
 # Print more verbose output (set logging level to INFO instead of default WARNING level).
-# verbose = False
+verbose = True
 
 # =========Start Global Config Option for Distributed L3 Router===============
 # Setting the "router_distributed" flag to "True" will default to the creation
@@ -74,7 +74,7 @@
 # with previous versions, the class name of a plugin can be specified instead
 # of its entrypoint name.
 #
-# service_plugins =
+service_plugins =
 # Example: service_plugins = router,firewall,lbaas,vpnaas,metering,qos
 
 # Paste configuration file
@@ -89,7 +89,7 @@
 
 # The strategy to be used for auth.
 # Supported values are 'keystone'(default), 'noauth'.
-# auth_strategy = keystone
+auth_strategy = keystone
 
 # Base MAC address. The first 3 octets will remain unchanged. If the
 # 4h octet is not 00, it will also be used. The others will be
@@ -357,14 +357,14 @@
 
 # ======== neutron nova interactions ==========
 # Send notification to nova when port status is active.
-# notify_nova_on_port_status_changes = True
+notify_nova_on_port_status_changes = True
 
 # Send notifications to nova when port data (fixed_ips/floatingips) change
 # so nova can update it's cache.
-# notify_nova_on_port_data_changes = True
+notify_nova_on_port_data_changes = True
 
 # URL for connection to nova (Only supports one nova region currently).
-# nova_url = http://127.0.0.1:8774/v2
+nova_url = http://ops-mono-node:8774/v2
 
 # Name of nova region to use. Useful if keystone manages more than one region
 # nova_region_name =
@@ -570,7 +570,7 @@
 
 # The messaging driver to use, defaults to rabbit. Other
 # drivers include qpid and zmq. (string value)
-# rpc_backend=rabbit
+rpc_backend=rabbit
 
 # The default exchange under which topics are scoped. May be
 # overridden by an exchange name specified in the
@@ -718,11 +718,12 @@
 # ===========  end of items for agent management extension =====
 
 [keystone_authtoken]
-auth_uri = http://127.0.0.1:35357/v2.0/
-identity_uri = http://127.0.0.1:5000
-admin_tenant_name = %SERVICE_TENANT_NAME%
-admin_user = %SERVICE_USER%
-admin_password = %SERVICE_PASSWORD%
+auth_uri = http://ops-mono-node:5000
+auth_url = http://ops-mono-node:35357
+auth_plugin = password
+project_name = service
+username = neutron
+password = password
 
 [database]
 # This line MUST be changed to actually run the plugin.
@@ -735,7 +736,7 @@
 # be set in the corresponding core plugin '.ini' file. However, it is suggested
 # to put the [database] section and its connection attribute in this
 # configuration file.
-connection = sqlite:////var/lib/neutron/neutron.sqlite
+connection = mysql+pymysql://neutron:password@localhost/neutrondb 
 
 # Database engine for which script will be generated when using offline
 # migration
@@ -774,6 +775,13 @@
 # pool_timeout = 10
 
 [nova]
+auth_url = http://ops-mono-node:35357
+auth_plugin = password
+region_name = Orange
+project_name = service
+username = nova
+password = password
+
 # Name of the plugin to load
 # auth_plugin =
 
@@ -984,7 +992,7 @@
 
 # The RabbitMQ broker address where a single node is used. (string value)
 # Deprecated group/name - [DEFAULT]/rabbit_host
-# rabbit_host = localhost
+rabbit_host = localhost
 
 # The RabbitMQ broker port where a single node is used. (integer value)
 # Deprecated group/name - [DEFAULT]/rabbit_port
@@ -1000,11 +1008,11 @@
 
 # The RabbitMQ userid. (string value)
 # Deprecated group/name - [DEFAULT]/rabbit_userid
-# rabbit_userid = guest
+rabbit_userid = openstack
 
 # The RabbitMQ password. (string value)
 # Deprecated group/name - [DEFAULT]/rabbit_password
-# rabbit_password = guest
+rabbit_password = password
 
 # The RabbitMQ login method. (string value)
 # Deprecated group/name - [DEFAULT]/rabbit_login_method
